.vscode/extensions.json

```
@@ -4,7 +4,7 @@
   "recommendations": [
     "EditorConfig.editorconfig",
     "esbenp.prettier-vscode",
-    "prisma.vscode-graphql",
+    "graphql.graphql",
```

I can't find this extension in the registry: https://marketplace.visualstudio.com/items?itemName=graphql.graphql
Are you sure this shouldn't be https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql ?
```suggestion
    "GraphQL.vscode-graphql",
```

---

I swear this worked yesterday 🤔